kiwix-desktop.pro

```diff
@@ -8,6 +8,10 @@ QT       += core gui network
 QT       += webenginewidgets
 QT       += printsupport
 
+#Determine if the host system is WSL based on the existence of "/proc/sys/fs/binfmt_misc/WSLInterop"
+WSL_CHECK = $$system(test -f /proc/sys/fs/binfmt_misc/WSLInterop && echo true || echo false)
+equals(WSL_CHECK, "true"): CONFIG += nostrip
+
```

Comment:
This comment is not so useful. We already see that we are checking existence of `/proc/sys/fs/binfmt_misc/WSLInterop` in the code. With a small variable renaming we can easily give the hint about what we are checking.

If we add comment, it should be why we do this, not what or how we are doing it.

I suggest:

```
# install script fails to detect if file is a binary or not on wsl and then try to strip all
# installed files, which obviously not work for all files. So don't strip on WSL.
ARE_WE_IN_WSL = $$system(test -f /proc/sys/fs/binfmt_misc/WSLInterop && echo true || echo false)
equals(ARE_WE_IN_WSL, "true"): CONFIG += nostrip
```

Reply:
LGTM, please do and squash everything in one commit so we can merge.